src/player-path.c

```diff
@@ -37,12 +37,12 @@
 /**
  * Maximum size around the player to consider in the pathfinder
  */
-#define MAX_PF_RADIUS 100
```

Comment:
With the current values for dungeon-wid and dungeon-hgt in lib/gamedata/constants.txt, a value of 397 is sufficient here.  Some platforms (at least one of the Nintendo systems and perhaps the DOS port) have tight memory constraints so avoiding increasing this more than is necessary has some use.